packages/block-library/src/group/block.json

```diff
@@ -15,6 +15,12 @@
 		"templateLock": {
 			"type": [ "string", "boolean" ],
 			"enum": [ "all", "insert", false ]
+		},
+		"layout": {
```

Comment:
Why is this being set here since this attribute comes from the block support?

Reply:
If we set the default in the block support, it would break back compat for existing blocks. Adding it as default here, together with the effect that's causing problems (#43760) in the edit function, ensures new blocks get the correct layout while existing blocks aren't changed.

If you have any ideas on how to do this better, I'm happy to explore them!